packages/template-retail-react-app/app/components/experience/image-tile/index.jsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import PropTypes from 'prop-types'
+import {AspectRatio, Image} from '@chakra-ui/react'
+
+/**
+ * This is a simple Image Tile component that can be used inside any Layout component
+ *
+ * @param imageProps
+ * @param ratio
+ * @returns {JSX.Element}
+ */
+const ImageTile = ({imageProps, ratio = 1, ...rest}) => {
+    return (
+        <AspectRatio ratio={ratio}>
+            <figure className={'photo-tile-figure'}>
+                <picture>
+                    <source srcSet={imageProps.src?.tablet} media="(min-width: 48em)" />
+                    <source srcSet={imageProps.src?.desktop} media="(min-width: 64em)" />
+                    <Image
+                        className={'photo-tile-image image-fluid'}
```

Comment:
Lets drop any utility classes if they aren't doing anything. Our convention currently in the template is to use inline styles (I think thats our convention). 

Also, I'm guessing the reason why the class name for the sfra component is called photo tile, is because that was it's initial implementation and then it was changed on the surface because someone didn't like the look of it in the page designer. 

So if we choose to use classnames for our components we should use the same name as our files. So in this case `image-tile` not `photo-tile`